sdk/python/packages/flet/mkdocs.yml

```diff
@@ -146,6 +146,9 @@ plugins:
         - source_dir: ../../examples
           target_url_path: examples
           include_exts: [".png", ".gif", ".svg"]
+        - source_dir: integration_tests
+          target_url_path: test-images
+          include_exts: [".png", ".gif", ".svg"]
```

Comment:
Can be removed when addressing https://github.com/flet-dev/flet/pull/5642#discussion_r2354299108.